ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -229,6 +251,218 @@ const triggerSet: TriggerSet<Data> = {
+          ja: '${dir1} / ${dir2}',
+          cn: '${dir1} / ${dir2}',
+          ko: '${dir1} / ${dir2}',
+        },
+      },
+    },
+    {
+      id: 'DSR Dragon\'s Rage',
+      // 63C4 Is Thordan's --middle-- action, thordan jumps again and becomes untargetable, shortly after the 2nd 6C34 action
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '63C4', source: 'King Thordan' }),
+      netRegexDe: NetRegexes.ability({ id: '63C4', source: 'Thordan' }),
+      netRegexFr: NetRegexes.ability({ id: '63C4', source: 'Roi Thordan' }),
+      netRegexJa: NetRegexes.ability({ id: '63C4', source: '騎神トールダン' }),
+      netRegexCn: NetRegexes.ability({ id: '63C4', source: '骑神托尔丹' }),
+      netRegexKo: NetRegexes.ability({ id: '63C4', source: '기사신 토르당' }),
+      condition: (data) => (data.phase === 'thordan' && (data.thordanJumpCounter = (data.thordanJumpCounter ?? 0) + 1) === 2),
+      delaySeconds: 0.5,
+      promise: async (data, matches) => {
+        // Select King Thordan
+        let thordanData = null;
+        thordanData = await callOverlayHandler({
+          call: 'getCombatants',
+          ids: [parseInt(matches.sourceId, 16)],
+        });
+
+        // if we could not retrieve combatant data, the
+        // trigger will not work, so just resume promise here
+        if (thordanData === null) {
+          console.error(`King Thordan: null data`);
+          return;
+        }
+        if (!thordanData.combatants) {
+          console.error(`King Thordan: null combatants`);
+          return;
+        }
+        const thordanDataLength = thordanData.combatants.length;
+        if (thordanDataLength !== 1) {
+          console.error(`King Thordan: expected 1 combatants got ${thordanDataLength}`);
+          return;
+        }
+
+        // Add the combatant's position
+        const thordan = thordanData.combatants.pop();
+        if (!thordan)
+          throw new UnreachableCode();
+        data.thordanDir = matchedPositionToDir(thordan);
+      },
+      infoText: (data, _matches, output) => {
+        // Map of directions
+        const dirs: { [dir: number]: string } = {
+          0: output.northwest!(),
+          1: output.north!(),
+          2: output.northeast!(),
+          3: output.east!(),
+          4: output.southeast!(),
+          5: output.south!(),
+          6: output.southwest!(),
+          7: output.west!(),
+          8: output.unknown!(),
+        };
+        return output.thordanLocation!({
+          dir: dirs[data.thordanDir ?? 8],
+        });
+      },
+      run: (data) => delete data.thordanDir,
+      outputStrings: {
+        north: Outputs.north,
+        northeast: Outputs.northeast,
+        east: Outputs.east,
+        southeast: Outputs.southeast,
+        south: Outputs.south,
+        southwest: Outputs.southwest,
+        west: Outputs.west,
+        northwest: Outputs.northwest,
+        unknown: Outputs.unknown,
+        thordanLocation: {
+          en: 'Boss: ${dir}',
```

Comment:
Small suggestion to put direction first, and call Thordan Thordan.

```suggestion
          en: '${dir} Thordan',
```